content/en/blog/2022/apisix.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Apache APISIX Integrates with OpenTelementry to Collect Tracing Data
+linkTitle: Apache APISIX-Opentelemetry Integration
+date: 2022-03-13
+author: Haochao Zhuang, Fei Han
+---
+
+This article introduces the Apache APISIX's `opentelemetry` plugin conceptand
+how to enable and deploy the plugin.
+
+## Background Information
+
+OpenTelemetry is an open source telemetry data acquisition and processing
+system. It not only provides various SDKs for application side telemetry data
+collection and reporting, but also provides data collection side for data
+receiving, processing and exporting. Export to any or more OpenTelemetry
+backends, such as Jaeger, Zipkin, and OpenCensus. You can view the list of
+plugins that have adapted the OpenTelemetry Collector in the
+[opentelemetry collector contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+library.
+
+![Architecture-Present](/img/apisix/architecture-present.png)
+
+## Plugin Introduction
+
+The `opentelemetry` plugin of Apache APISIX implements Tracing data collection
+based on OpenTelemetry native standard (OTLP/HTTP), and sends it to
```

Comment:
```suggestion
and sends it to
```
I think this is not relevant and confusing piece of information

Reply:
Solved.